src/_gettsim/transfers/rente.py

```diff
@@ -914,22 +1400,62 @@ def ges_rente_vorauss_frauen(
         weiblich
         and ges_rente_wartezeit_15
         and y_pflichtbeitr_ab_40 > ges_rente_params["rente_für_frauen_pflichtbeitr_y"]
-        and geburtsjahr < ges_rente_params["abolishment_cohort_rente_für_frauen"]
+        and geburtsjahr < ges_rente_params["first_birthyear_without_rente_für_frauen"]
     )
 
     return out
 
 
-def _ges_rente_vorauss_arbeitsl(
+@dates_active(end="2006-12-31", change_name="_ges_rente_vorauss_arbeitsl")
+def _ges_rente_vorauss_arbeitsl_ohne_2007_reform(
+    arbeitsl_1y_past_585: bool,
+    ges_rente_wartezeit_15: bool,
+    pflichtbeitr_8_in_10: bool,
+) -> bool:
+    """Eligibility for Altersrente für Arbeitslose (pension
+    for unemployed).
+
+    Wartezeit 15 years, 8 contribution years past 10 years, being unemployed for at
+    least 1 year after age 58 and 6 months.
+
+    Parameters
+    ----------
+    arbeitsl_1y_past_585
+        See basic input variable
+        :ref:`arbeitsl_1y_past_585 <arbeitsl_1y_past_585>`.
+    ges_rente_wartezeit_15
+        See :func:`ges_rente_wartezeit_15`
+    pflichtbeitr_8_in_10
+        See basic input variable :ref:`pflichtbeitr_8_in_10 <pflichtbeitr_8_in_10>`.
+
+    Returns
+    -------
+    Eligibility as bool.
+
+    """
+
+    out = arbeitsl_1y_past_585 and ges_rente_wartezeit_15 and pflichtbeitr_8_in_10
+
+    return out
+
+
+@dates_active(
+    start="2007-01-01", end="2017-12-31", change_name="_ges_rente_vorauss_arbeitsl"
+)
+def _ges_rente_vorauss_arbeitsl_mit_2007_reform(
     arbeitsl_1y_past_585: bool,
     ges_rente_wartezeit_15: bool,
     pflichtbeitr_8_in_10: bool,
     birthdate_decimal: float,
     ges_rente_params: dict,
 ) -> bool:
-    """Determining the eligibility for Altersrente für Arbeitslose (pension
-    for unemployed. Wartezeit 15 years, 8 contributionyears past 10 years, being
-    at least 1 year unemployed after age 58 and 6 months.
+    """Eligibility for Altersrente für Arbeitslose (pension
+    for unemployed).
+
+    Wartezeit 15 years, 8 contributionyears past 10 years, being
```

Comment:
Can you watch the line breaks a bit better, please? Alt + Q does it for me in VS Code, happy to look into the settings.